easybuild/easyconfigs/p/PyTorch/PyTorch-1.2.0_fix_torchvision_test.patch

```diff
@@ -0,0 +1,36 @@
+pytorch #26608
```

Comment:
@branfosj please include the full URL (and fix the patch checksum afterwards)?

Reply:
done